src/main/java/dwbh/api/domain/input/UserGroupAndVisibleMemberListInput.java

```diff
@@ -66,7 +66,7 @@ public boolean isVisibleMemberList() {
    * @param groupId the id of the group
    * @since 0.1.0
    */
-  public UserGroupInput(UUID userId, UUID groupId) {
+  public UserGroupAndVisibleMemberListInput(UUID userId, UUID groupId) {
```

Comment:
Is this constructor used ? If it's not, then remove it.

Reply:
Done